tests/Test/AbstractFixerTestCase.php

```diff
@@ -201,6 +201,8 @@ final public function testFixerDefinitions(): void
                 static::assertMatchesRegularExpression('/^[a-z_]+[a-z]$/', $option->getName(), sprintf('[%s] Option %s is not snake_case.', $fixerName, $option->getName()));
             }
         }
+
+        static::assertIsInt($this->fixer->getPriority());
```

Comment:
is it needed? like, shouldnt this be checked with strong typehinting of ``getPriority(): int`?

Reply:
Yeah it kept being reported as not covered.
While the method signature enforces the return type it still won't show up as failed in the tests when not returning an int until the method is actual called.
Alternatively we could tag the various priority test methods in the FixerFactoryTest to state that these are testing the getPriority method and see if that reflects nicely in the coverage reports.

Reply:
> Alternatively we could tag the various priority test methods in the FixerFactoryTest to state that these are testing the getPriority method and see if that reflects nicely in the coverage reports.

This solution would be more accurate indeed. Or maybe in `IntegrationTest`.

The current solution has no real value because it only checks what native type declarations already does. Returning a wrong value would be detected by PHPStan anyway. Also, PHPStan could even detect that this assertion is redundant, but I think we need to enable an extension or rise analyze level.

Testing priority only makes sense through priority integration tests, i.e. testing the outcome instead of the implementation detail.